### PR TITLE
Added booking controller

### PR DESCRIPTION
**Pull request by ElisaVanderPerre**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -11,3 +11,11 @@
 // Your CSS partials
 @import "components/index";
 @import "pages/index";
+
+.notice {
+  margin: 0px;
+}
+
+.container {
```
Comment (Iskim91) [inline app/assets/stylesheets/application.scss]: this container will affect all the containers in different pages?

File: app/assets/stylesheets/components/_card.scss [hunk 2]
```diff
@@ -43,3 +43,7 @@
   top: -20px;
   width: 40px;
 }
+
+.card {
```
Comment (Iskim91) [inline app/assets/stylesheets/components/_card.scss]: will this card will clash with the existing card class

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (ElisaVanderPerre) [reply app/assets/stylesheets/application.scss]: yes, but I checked it! It should normally work for all pages.

File: app/assets/stylesheets/components/_card.scss (hunk 2 above)
Comment (ElisaVanderPerre) [reply app/assets/stylesheets/components/_card.scss]: it didn't change any of you cards, because you used the card-trip class

**Pull request merged**